### PR TITLE
1 user visits app

### PR DESCRIPTION
**Pull request by adamcaron**

### CODE REVIEW

**Comment by adamcaron** [comment]
#1

**Comment by adamcaron** [comment]
close #13 close #12 close #1

File: spec/features/user_visits_app_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.feature "User visits app" do
+
+  scenario "sees all the tasks" do
+    visit root_path
+
+    expect(page).to have_css('div.tasks', text: "Tasks")
```
Comment (biglovisa) [inline spec/features/user_visits_app_spec.rb:8]: Nice, remind me to scope these assertions with CSS selectors when I add the styling!

**Comment by biglovisa** [comment]
Looks good! Merging

**Pull request merged**